datarequest.py

```diff
@@ -240,10 +247,14 @@ def transform_title(row):
 
 @api.make()
 def api_datarequest_schema_get(ctx, schema_name):
+    return datarequest_schema_get(ctx, schema_name)
+
+
+def datarequest_schema_get(ctx, schema_name):
     """Get schema and uischema of a datarequest form
 
        Arguments:
-       schema_name -- Name of schema.
+       schema_name -- Name of schema
```

Comment:
Sphinx docstyle: https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html

```suggestion
       :param schema_name: Name of JSON schema
```